azure-pipelines.yml

```diff
@@ -159,8 +159,8 @@ stages:
           test_command: 'pytest -p no:warnings --doctest-rst -m "not mpl_image_compare" --pyargs sunpy'
           submodules: false
           targets:
-            - wheels_cp3[789]-manylinux*x86_64
-            - wheels_cp3[789]-macosx*
+            - wheels_cp3[78910]-manylinux*x86_64
+            - wheels_cp3[78910]-macosx*
```

Comment:
This just seems very wrong.

Reply:
yeah this doesn't work, this regex only matches one character.

When I did Astropy recently I couldn't find a functioning regex so I split them into their own jobs. The other option is use the env vars that cibuildwheel provides and then just match all Pythons here.

Reply:
This is the real millenium bug.

Reply:
Separate targets it is.